### PR TITLE
Update relationships.md

### PR DESCRIPTION
**Pull request by wanghengxn**
The test results are as described above. 

jhipster -V
INFO! Using JHipster version installed globally
6.10.5

### CODE REVIEW

**Comment by wanghengxn** [comment]
I found that there are too many things wrong with this document. Maybe you should find someone to check it out.

**Review by mshima — APPROVED** [review]

File: pages/jdl/relationships.md [hunk 1]
```diff
@@ -122,12 +122,20 @@ relationship OneToOne {
 
 Note that this example is the same as:
 ```jdl
+relationship OneToOne {
+  A{b} to B{a}
+}
+```
+Not specifying an injected field is the short form of using having an bidirectional relationship.
+
+another example:
```
Comment (MathieuAA) [inline pages/jdl/relationships.md:131]:
```suggestion
Another example:
```

File: pages/jdl/relationships.md [hunk 2]
```diff
@@ -122,12 +122,20 @@ relationship OneToOne {
 
 Note that this example is the same as:
 ```jdl
+relationship OneToOne {
+  A{b} to B{a}
+}
+```
+Not specifying an injected field is the short form of using having an bidirectional relationship.
```
Comment (MathieuAA) [inline pages/jdl/relationships.md:129]:
```suggestion
Not specifying an injected field is the short form of having a bidirectional relationship.
```

**Review by MathieuAA — CHANGES_REQUESTED** [review]
Seems like some changes were made to the generation part and this doc hasn't been up to date... :(

**Pull request merged**